components/collection/activity/ActivityChart.vue

```diff
@@ -43,16 +46,26 @@ const listEvents = computed(() => {
 })
 
 const chartData = computed(() => {
-  const buyBins = bin(buyEvents.value, { days: 1 })
-  const listBins = bin(listEvents.value, { days: 1 })
+  const buyBinSize = getBinSizeForRange({
+    firstTimestamp: buyEvents.value[0].timestamp,
+    lastTimestamp: buyEvents.value[buyEvents.value.length - 1].timestamp,
+    datasetLength: buyEvents.value.length,
+  })
+  const listBinSize = getBinSizeForRange({
+    firstTimestamp: listEvents.value[0].timestamp,
+    lastTimestamp: listEvents.value[listEvents.value.length - 1].timestamp,
+    datasetLength: listEvents.value.length,
+  })
+  const buyBins = bin(buyEvents.value, buyBinSize)
+  const listBins = bin(listEvents.value, listBinSize)
 
   const binnedBuyEvents = buyBins.map(({ timestamp, value }) => [
     new Date(timestamp),
-    displayValue(value),
+    displayValue(value, decimals.value),
   ])
   const binnedListEvents = listBins.map(({ timestamp, value }) => [
     new Date(timestamp),
-    displayValue(value),
+    displayValue(value, 10),
```

Comment:
Why hard coding?
```suggestion
    displayValue(value, decimals.value),
```

Reply:
absolutly no reason. thanks for catching it

Reply:
fixed